lib/jsonapi-response.js

```diff
@@ -0,0 +1,102 @@
+const attributesMap = require('../fixtures/attributes');
+const relationshipsMap = require('../fixtures/relationships');
+const config = require('../config');
```

Comment:
I'm generally more included to pass config in to modules/functions as a parameter. The same going for any stateful dependencies (like elasticsearch client in our case).

Requiring config in a module actually creates a dependency on whatever config module we're using to configure the app (in our case `rc`) and makes it a little more difficult to test in isolation.

In this case it's not a massive problem, we can just require the config in the test to grab the `rootUrl` so we can make our assertions. Imagine for example if the module used some secrets to connect to stripe or something - in an integration test you probably don't want whatever credentials you have in config for development, you probably want to pass it config for connecting to the test Stripe server and using test credentials.

Put it this way, if you were to extract this module and publish it to npm, how would you allow consumers to set the root URL - you probably wouldn't enforce use of a particular config module and would probably instead pass it in as a parameter to the function.

https://github.com/TheScienceMuseum/collectionsonline/blob/master/lib/transforms/search-results-to-jsonapi.js#L4